pymc/variational/opvi.py

```diff
@@ -829,23 +702,15 @@ def __init__(
         params=None,
         random_seed=None,
         model=None,
-        local=False,
-        rowwise=False,
         options=None,
         **kwargs,
     ):
-        if local and not self.supports_batched:
-            raise LocalGroupError("%s does not support local groups" % self.__class__)
-        if local and rowwise:
-            raise LocalGroupError("%s does not support local grouping in rowwise mode")
         if isinstance(vfam, str):
             vfam = vfam.lower()
         if options is None:
             options = dict()
         self.options = options
         self._vfam = vfam
-        self._local = local
-        self._batched = rowwise
         self._rng = at_rng(random_seed)
```

Comment:
I guess this change conflicts with the main branch, a rebase or merge should help to fix that